### PR TITLE
:Memo: Fix documentation errors and also docstring format

### PR DESCRIPTION
**Pull request by ArWeHei**
There are still some files in the old google doctoring format, but there are no more errors thrown when building the documentation. See #78 #104

### CODE REVIEW

**Review by jhaux — COMMENTED** [review]
Nice, that looks really good! The only thing I noticed is the removal of the `BaseTrainer` class (`edflow/iterators/tf_trainer`). I personally do not use it, but maybe this introduces some breaking changes?
@pesser and @theRealSuperMario maybe you can comment as well?

File: edflow/iterators/tf_evaluator.py [hunk 1]
```diff
@@ -13,53 +13,6 @@
 P = ProjectManager()
 
 
-class BaseEvaluator(HookedModelIterator):
```
Comment (pesser) [inline edflow/iterators/tf_evaluator.py:16]: removed deprecated BaseEvaluator

File: edflow/iterators/tf_iterator.py [hunk 2]
```diff
@@ -2,162 +2,6 @@
 from edflow.iterators.model_iterator import PyHookedModelIterator
 
 
-class HookedModelIterator(object):
```
Comment (pesser) [inline edflow/iterators/tf_iterator.py:5]: removed deprecated HookedModelIterator

File: edflow/iterators/tf_trainer.py [hunk 3]
```diff
@@ -18,161 +18,6 @@
 P = ProjectManager()
 
 
-class BaseTrainer(HookedModelIterator):
```
Comment (pesser) [inline edflow/iterators/tf_trainer.py:21]: remove deprecated BaseTrainer

File: edflow/iterators/tf_trainer.py [hunk 4]
```diff
@@ -257,17 +102,20 @@ def create_train_op(self):
...
+        self.lr_decay_end = self.config.get(
+            "lr_decay_end", self.config.get("num_steps")
+        )
 
         self.lr = lr = make_linear_var(
             self.global_step,
             self.lr_decay_begin,
             self.lr_decay_end,
             self.initial_lr,
-            0.0,
```
Comment (pesser) [inline edflow/iterators/tf_trainer.py:268]: changed default behavior to not decay to zero but keep it constant

File: edflow/iterators/tf_trainer.py [hunk 5]
```diff
@@ -394,17 +242,20 @@ def create_train_op(self):
...
```
Comment (pesser) [inline edflow/iterators/tf_trainer.py:405]: changed default behavior to not decay to zero but keep it constant

**Review by pesser — APPROVED** [review]
wow, thanks for all the work!

**Pull request merged**